scripts/mocha2md.js

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs'
```

Comment:
This little script allows us to generate a [markdown job summary](https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/) from a mocha JSON report. We should be able to re-use it for the other mocha-based test runs: integration, package, services, etc
You can see an examples of the output here:

- https://github.com/badges/shields/actions/runs/2895774184
- https://github.com/badges/shields/actions/runs/2895785327